articles/rules-are-dangerous.md

```diff
@@ -0,0 +1,53 @@
+
+Please, do not get me wrong: I like the Agile Manifesto, the SOLID Principles, Clean Code, Martin Fowler in general and even his article about the Page Object Pattern. Those are all practices I appreciate a lot and that influenced my way of thinking and working. But still all these things also come with the danger of undesired side-effects, if just following them stupidly or in a very dogmatic way.
+
+## The Problems with Rules
+
+The problems I often see in projects with such rules:
+* Applying rules in the wrong context might not have the desired effect at all
+* Using rules stricter than meant to be, causes adverse effects
+* Applying rules without really understanding them can result in more harm to the project than benefits
+* Some rules might be treated as god given, just because it has been defined in some famous article by some clever guy or in that holy Scrum bible or the process manual written by the higher management. This causes that people will not argue about these rules anymore about why, when and how some rules should be followed exactly. This can cause destructive discussion culture and lead to bad decisions without real arguments.
+
+## Conclusion and Recommendations
+
+Here are my recommendations on how to mitigate the problems with such rules in your projects:
+
+* We all must be always willing to explain ourselves, why we do something like that and what are the arguments to apply a rule in which context. Something like “just because our process manual says so” or “because I’ve read it in that article” is usually not a good answer.
+* Don’t try to replace critical thinking and adaptive methods by exact and rigid rules that people must follow strictly. 
+* But do not read my recommendations here in a dogmatic way. Which would be: “Rolf said rules are bad and harmful, so let’s trash all our rules in our project and don’t follow the scrum guide, our Definition of Done, Clean Code and similar things anymore”. This was not my message at all! 
+* Instead: Discuss, write down and live these rules in your Team, but see it more as a guide than exact rules to be 100% precisely followed in any case.
+* Note down not just the rules you use in your project but also the arguments why a rule should be applied. 
+* Be open to discuss and reexplain rules in your projects with your team members and to adjust the rules to changes in the context (e.g. growing software, growing team, shorter release cycles, different team members, new ideas, etc.)
+* Be critical to existing rules, but also respect and try to understand them in a first place.
+
+Also remember this when reading all the other articles in this book from my brilliant coworkers! 
+There are many great rules and ideas in these articles that will help you a lot when applied wisely in the right context.
+
+## Further Readings
```

Comment:
Bitte Kapitel Further Readings in neues File "rules-are-dangerous.resources.md" verschieben und am Schluss des Artikels folgendermassen verlinken: 

{% include "./rules-are-dangerous.resources.md" %}

Reply:
Kann ich tun, auch wenn mir der Sinn dieser Stückelung mit den Includes nicht ganz klar ist.

Funktioniert das mit der Stückelung und den includes tatsächlich mit dem gitbook Tooling?

Interessant.

Reply:
Done